CacheRefresh.php

```diff
@@ -2,53 +2,39 @@
 
 namespace SofaScore\Purgatory;
 
+use Exception;
+use RuntimeException;
 use SofaScore\Purgatory\Mapping\Loader\LoaderInterface;
 use SofaScore\Purgatory\Mapping\MappingCollection;
-use SofaScore\Purgatory\Mapping\MappingValue;
 use Symfony\Component\ExpressionLanguage\ExpressionLanguage;
 use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
 
 class CacheRefresh
 {
-    public const PRIORITY_DEFAULT = '0';
-    public const ROUTE_TAG = 'route';
+    private const PRIORITY_DEFAULT = '0';
+    private const ROUTE_TAG = 'route';
 
-    /**
-     * @var MappingCollection
-     */
-    protected $mappings;
+    private ?MappingCollection $mappings = null;
 
-    /**
-     * @var LoaderInterface
-     */
-    protected $mappingsLoader;
+    private LoaderInterface $mappingsLoader;
 
-    /**
-     * @var PropertyAccessorInterface
-     */
-    protected $propertyAccessor;
+    private PropertyAccessorInterface $propertyAccessor;
 
-    /**
-     * @var ExpressionLanguage
-     */
-    protected $expressionLanguage;
+    private ExpressionLanguage $expressionLanguage;
 
     public function __construct(LoaderInterface $mappingsLoader, PropertyAccessorInterface $propertyAccessor)
     {
         $this->mappingsLoader = $mappingsLoader;
         $this->propertyAccessor = $propertyAccessor;
 
-        if (!class_exists('Symfony\Component\ExpressionLanguage\ExpressionLanguage')) {
-            throw new \RuntimeException('Unable to use expressions as the Symfony ExpressionLanguage component is not installed.');
+        if (!class_exists(ExpressionLanguage::class)) {
```

Comment:
@shule727 Slobodno makni kompletno ovu provjeru pošto je expression language komponenta postavljena kao dependency kroz Composer.